docs/resource-providers/rds-providers.md

```diff
@@ -0,0 +1,126 @@
+# Relational Database Service Resource Providers
+
+The relational database resource provider specifically tightly couples the lifetime of the RDS
+Resources created with that of the EKS Cluster. It should only be used with a so-called management
```

Comment:
Let's say that "production usage should be using the centralized/management cluster pattern.